build/install-dotnet.yml

```diff
@@ -0,0 +1,14 @@
+steps:
+- task: UseDotNet@2
+  inputs:
+    packageType: 'sdk'
+    version: '3.1.x'
+    installationPath: 'C:\Program Files\dotnet'
+    performMultiLevelLookup: true
+- task: PowerShell@2
+  displayName: 'dotnet --info'
```

Comment:
added this task for future debugging... i couldn't tell if anything had changed wrt the install logic b/c there wasn't enough logging to tell me how it used to work